client-react/src/pages/app/change-app-plan/ChangeAppPlan.tsx

```diff
@@ -146,6 +112,128 @@ export const ChangeAppPlan: React.SFC<ChangeAppPlanProps> = props => {
+    );
+  };
+
+  const getCurrentPlanDetails = () => {
+    return (
+      <>
+        <Stack style={sectionStyle}>
+          <h4 className={labelSectionStyle}>{t('changePlanCurrentPlanDetails')}</h4>
+        </Stack>
+
+        <ReactiveFormControl id="currentAppServicePlan" label={t('appServicePlan')}>
+          <div tabIndex={0} aria-label={t('appServicePlan') + getPlanName(currentServerFarm)}>
+            {getPlanName(currentServerFarm)}
+          </div>
+        </ReactiveFormControl>
+      </>
+    );
+  };
+
+  const getDestinationPlanDetailsHeader = () => {
+    return (
+      <Stack style={{ marginTop: '50px' }}>
+        <h4 className={labelSectionStyle}>{t('changePlanDestPlanDetails')}</h4>
+      </Stack>
+    );
+  };
+
+  const getPlanTypeRadioButton = (formProps: FormikProps<ChangeAppPlanFormValues>) => {
+    return (
+      <>
+        {Url.getFeatureValue(CommonConstants.FeatureFlags.enableFunctionsDynamicToPremium) === 'true' && (
+          <div className={planTypeStyle}>
+            <ReactiveFormControl id="planType" label={t('planType')}>
+              <RadioButtonNoFormik
+                id="planType"
+                ariaLabelledBy={t('planType')}
+                defaultSelectedKey={formProps.values.serverFarmInfo.newPlanInfo.tier}
+                options={[
+                  { key: ChangeAppPlanTierTypes.Dynamic, text: t('consumptionPlan') },
+                  { key: ChangeAppPlanTierTypes.ElasticPremium, text: t('functionPremiumPlan') },
+                ]}
+                onChange={(o, e) => e && handleTierChange(e.key as ChangeAppPlanTierTypes)}></RadioButtonNoFormik>
+            </ReactiveFormControl>
+          </div>
+        )}
+      </>
+    );
+  };
+
+  const handleTierChange = (planTier: ChangeAppPlanTierTypes) => {
+    //TODO: (stpelleg): reload plans
+  };
+
+  const getDestinationAppServicePlan = (formProps: FormikProps<ChangeAppPlanFormValues>) => {
+    return (
+      <ReactiveFormControl id="destinationAppServicePlan" label={t('appServicePlan')} required={true}>
+        <CreateOrSelectPlan
+          subscriptionId={subscriptionId}
+          isNewPlan={formProps.values.serverFarmInfo.isNewPlan}
+          newPlanInfo={formProps.values.serverFarmInfo.newPlanInfo}
+          existingPlan={formProps.values.serverFarmInfo.existingPlan}
+          options={serverFarmOptions}
+          resourceGroupOptions={rgOptions}
+          onPlanChange={info => {
+            onPlanChange(formProps, info);
+          }}
+          serverFarmsInWebspace={serverFarms}
+          hostingEnvironment={hostingEnvironment}
+        />
+      </ReactiveFormControl>
+    );
+  };
+
+  const getCurrentResourceGroup = (formProps: FormikProps<ChangeAppPlanFormValues>) => {
+    return (
+      <ReactiveFormControl id="currentResourceGroup" label={t('resourceGroup')}>
+        <div tabIndex={0} aria-label={t('resourceGroup') + getSelectedResourceGroupString(formProps.values.serverFarmInfo, t)}>
```

Comment:
Add a space between the label and the resource group name to avoid screen reader mispronunciation.
```suggestion
        <div tabIndex={0} aria-label={`${t('resourceGroup')} ${getSelectedResourceGroupString(formProps.values.serverFarmInfo, t)}`}>
```